test/extended/etcd/etcd_storage_path.go

```diff
@@ -712,6 +713,10 @@ func getFromEtcd(kv etcdv3.KV, path string) (*metaObject, error) {
 		if err := json.Unmarshal(value, metaObj); err != nil {
 			return nil, err
 		}
+	case isCBOR(value):
```

Comment:
Any CBOR in storage should have the prefix 0xd9d9f7. It's not sufficient to check if it can be understood by a CBOR decoder because there can be byte sequences that are valid in more than one protocol.